startupscript/setup-bashrc.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
It would be great if we could pull in all of the commentary from:

https://github.com/verily-src/mc-terra-authz-wsm/blob/60048df8e901c6bf35b3be96fa0969380ba15236/service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh#L128

or we set somewhere else as the "canonical" documentation around these environment variables. Currently it says:

```
# Set variables into the .bash_profile such that they are available
# to terminals, notebooks, and other tools
#
# We have new-style variables (eg GOOGLE_CLOUD_PROJECT) which are set here
# and CLI (terra app execute env).
# We also support a few variables set by Leonardo (eg GOOGLE_PROJECT).
# Those are only set here and NOT in the CLI as they are intended just
# to make porting existing notebooks easier.

# Keep in sync with terra CLI environment variables:
# https://github.com/DataBiosphere/terra-cli/blob/14cf51dd809573c7ae9a3ef10ddd427fa057cb8f/src/main/java/bio/terra/cli/app/CommandRunner.java#L88

# *** Variables that are set by Leonardo for Cloud Environments
# (https://github.com/DataBiosphere/leonardo)
```

The pointer to the terra CLI repository should be updated as well.